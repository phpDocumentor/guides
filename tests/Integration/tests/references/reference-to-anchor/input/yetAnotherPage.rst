
================
Yet Another Page
================

Lorem Ipsum

FlexForms
=========

:ref:`FlexForms <flexforms>`
