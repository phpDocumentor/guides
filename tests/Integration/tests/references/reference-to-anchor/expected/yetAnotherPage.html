<!-- content start -->
    <div class="section" id="yet-another-page">
            <h1>Yet Another Page</h1>

    <p>Lorem Ipsum</p>

            <div class="section" id="flexforms">
            <h2>FlexForms</h2>

    <p><a href="index.html#flexforms">FlexForms</a></p>

    </div>
    </div>
<!-- content end -->
